pkg/process/checks/net.go

```diff
@@ -408,11 +408,10 @@ func batchConnections(
 				continue
 			}
 
-			//nolint:revive // TODO(NET) Fix revive linter
```

Comment:
These files are Networks related but have Process team code ownership which is why Processes got pulled into code review